aiogram/types/user.py

```diff
@@ -40,3 +41,17 @@ def full_name(self) -> str:
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def url(self) -> str:
+        return f"tg://user?id={self.id}"
```

Comment:
Is preffered to use `create_tg_link` function which gives an identical result
```python
create_tg_link("user", id=self.id)
```
Can be imported from:
```python
from aiogram.utils.link import create_tg_link
```